subsys/nrf_security/src/ssf_secdom/CMakeLists.txt

```diff
@@ -4,8 +4,36 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
+if(CONFIG_SSF_V_2)
+target_sources(${mbedcrypto_target}
```

Comment:
Missing indentation?

Also, why are we not using `zephyr_` cmake functions for adding sources and include dirs?

Reply:
It's intentionally missing to make the diff smaller when we drop support for SUIT and delete the if(CONFIG_SSF_V_2).

target_sources is used because that is what the existing ssf_secdom build scripts used.

I don't know why they use target_sources.

Reply:
Do we care about a small diff? (not being sarcastic here) In case we don't then you don't see any issues with using `zephyr_` functions?

Reply:
fixed. I kept the SSFv1 build as-is to avoid obscure regressions.